apps/storybook/src/FileUpload.stories.tsx

```diff
@@ -74,3 +77,54 @@ export const WrappingInput: Story<FileUploadProps> = (args) => {
 WrappingInput.args = {
   dragContent: 'Drop file to upload',
 };
+
+export const SingleFileUpload: Story<FileUploadProps> = (args) => {
+  return (
+    <FileUpload
+      {...args}
+      onFileDropped={(files) => {
+        action(`${files.length} files uploaded`)();
+      }}
+    >
+      <FileUploadCard />
+    </FileUpload>
+  );
+};
+
+export const SingleFileUploadCustom: Story<FileUploadProps> = (args) => {
+  const inputRef = React.useRef<HTMLInputElement>(null);
+  const [files, setFiles] = React.useState<File[]>([]);
+  const fileInputId = React.useId();
+
+  return (
+    <FileUpload
```

Comment:
Have you tested whether it works without `FileUpload`? we need to make it work

also, the `files`/`onFilesChange` props should be added to `FileUpload`. usually in compound components, the component at the very top should control the state

Reply:
Outside of the dragging functionality, this component works without `FileUpload`. I changed the CSS for `iui-file-card` and the newly-created `iui-file-card-empty` so that it matches the previous screenshots when outside of `iui-content`. However, I have some CSS issues. 

When `iui-file-card` is wrapped by `iui-content`, extra padding from `iui-content` is added to the padding from `iui-file-card`. This also leads to issues with `focus-within` when comparing the two structures. 

![image](https://user-images.githubusercontent.com/78564221/221994829-967ef87d-1694-4fd9-892a-3892939573e6.png)

I've added the two types of HTML structures for `FileUploadCard` (One standalone and one wrapped by the `FileUpload` component) in the demo pages until I can resolve this issue.
	
As for the other issue, how would I pass the `files` and `onFilesChanged` props from `FileUpload` to `FileUploadCard` if the `children` of `FileUpload` isn't always a `FileUploadCard` component?

Reply:
I was able to get the sizing of the two cards to match by adding a `margin` to `iui-file-card` and removing it if it had a parent of `iui-content`. I have also fixed the double-outline issue, but since the outline goes around the `margin` now, the `focus-within` still just isn't quite right :/

![image](https://user-images.githubusercontent.com/78564221/222846127-039b06a9-ae3d-46bb-b11d-a10576da203d.png)

I've updated the react screenshots and I will revert the html demo page and upload the css screenshots once this issue is resolved